process/copy.go

```diff
@@ -10,14 +10,16 @@ import (
 
 /*
 Copy processes data by copying it. The processor supports these patterns:
-	JSON:
-	  	{"hello":"world"} >>> {"hello":"world","goodbye":"world"}
-	from JSON:
-  		{"hello":"world"} >>> world
-	to JSON:
-  		world >>> {"hello":"world"}
+
+		JSON:
+		  	{"hello":"world"} >>> {"hello":"world","goodbye":"world"}
+		from JSON:
+	  		{"hello":"world"} >>> world
+		to JSON:
+	  		world >>> {"hello":"world"}
```

Comment:
docs: this indent seems to have an extra tab compared to comments for other processors? is this correct? 🤔

Reply:
Whoops, no this was accidental - removed this extra indentation with the latest fixup. Thanks or catching this!